content/blog/2022/2022-05-12-Google-Tides.md

```diff
@@ -0,0 +1,30 @@
++++
+author = "Zala Gruden"
+date = "2022-05-12"
+draft = false
+title = "We have launched a new project on the topic of AI in schools"
+type = "blog"
+thumbImage = "/blog_img/2022/2022-05-12-google-tides.png"
+frontPageImage = "/blog_img/2022/2022-05-12-google-tides.png"
+blog = ["Google", "Tides Foundation", "project", "visualization"]
+shortExcerpt = "Find out more about our new project about AI in Schools."
+longExcerpt = "The website to our latest project has just been launche. Read more about the project in the blogpost and Check out the site to find out more about the prepaired activities."
++++
+
+{{< window-screenshot src="/blog_img/2022/2022-05-12-google-tides.png" >}} 
+
+We have successfully launched a new project about AI and data science in elementary and high schools. 
+
+
+AI is the technology of the century, yet too few kids, and education professionals, know about its concepts, ideas and applications. 
+The project, funded by **Google.org** and **Tides Foundation**, is aimed at bringing AI to teachers and kids by designing hands-on workshops by focusing on conceptual learning and applications. 
+
+During the course of the project, between **January 31st 2022** and **July 31st 2023**, we are expecting to carry out several activities whose long term outcome is expected to improve overall education about AI in Slovenia. By the end of the project, we will have executed ten two hour workshops, designed for primary and secondary schools to enhance the current curricula with AI and machine learning. We will also train 24 teacher ambassadors and five-hundred elementary and high-school kids in the first round of workshops and two hundred teachers in the train-the-trainer workshops and over one thousand kids in the second round of workshops. 
+This two-year project has further reaching goals. We will establish working groups and communication channel to continuously support the training of AI and machine learning of educators. 
```

Comment:
further reaching --> further-reaching
We will establish working groups and communication channel to continuously support the training of AI and machine learning of educators. --> We will establish working groups and communication channels to continuously support AI and machine learning training for the educators.